resources/sdk/purecloudkotlin/templates/README.mustache

```diff
@@ -45,7 +45,7 @@ val users = usersApi.getUsers(null, null, null, null, null, null, null)
 
 For user applications, the consuming application must complete an implicit, auth token, or SAML2 Bearer OAuth flow to get an access token outside the scope of the SDK. Once an access token is obtained, it should be set on the SDK via constructing a new ApiClient instance (use `withAccessToken(token: String)`). For more information about authenticating with OAuth, see the Developer Center article [Authorization](https://developer.mypurecloud.com/api/rest/authorization/index.html). For more information about SAML2 Bearer Oauth flow view the example below 
 
-## Authentication with SAML2Bearer token
+#### Authentication with SAML2Bearer token
 
 ```{"language":"kotlin"}
 val clientId = System.getenv("PURECLOUD_CLIENT_ID")
```

Comment:
While you're making these edits, can you rename the env vars in the readme files to use `GENESYS_CLOUD`? e.g.  `GENESYS_CLOUD_CLIENT_ID`